lib/p2p/packets/types/HelloPacket.ts

```diff
@@ -0,0 +1,25 @@
+import Packet, { PacketHeader, MessageType } from '../Packet';
+import PacketType from '../PacketType';
+
+type HelloPacketBody = {
+  version: string;
+  nodeKey: string;
+  listenPort: number;
+  pairs: string[];
+};
+
+class HelloPacket extends Packet {
+  public type: PacketType = PacketType.HELLO;
+  public messageType: MessageType = MessageType.UNILATERAL;
```

Comment:
Why the change from a packet with a response to a unilateral one?

Reply:
The previous design I did was buggy.  If the node will receive `Hello` before it sent an `Hello` , and the generic req/res mechanism will be applied, then a new `Hello` will never be received. This scenario can be easily verified by playing with the intervals (it didn't happen just because we're still testing multiple nodes on one machine only). And because anyway `Hello` should be sent voluntarily, without any prior request, I wasn’t a good fit for this pattern anyway.

Reply:
Cool, just so I'm clear - upon connection both nodes will send Hellos and once both Hellos have been received the connection can start being used.

Reply:
Yep. But we might add more async steps to the handshake later.